app-text/aspell/aspell-0.60.6.1.recipe

```diff
@@ -12,9 +12,10 @@ more than one Aspell process is open at once."
 HOMEPAGE="http://aspell.net"
```

Comment:
Could you please add a trailing slash for us at the end of [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage)?
```diff
-HOMEPAGE="http://aspell.net"
+HOMEPAGE="http://aspell.net/"
```
Thanks!